x/evidence/internal/types/params.go

```diff
@@ -0,0 +1,60 @@
+package types
+
+import (
+	"time"
+
+	"github.com/cosmos/cosmos-sdk/x/params"
+
+	"gopkg.in/yaml.v2"
+)
+
+// DONTCOVER
+
+// Default parameter values
+const (
+	DefaultParamspace     = ModuleName
+	DefaultMaxEvidenceAge = 60 * 2 * time.Second
+)
+
+// Parameter store keys
+var (
+	KeyMaxEvidenceAge = []byte("MaxEvidenceAge")
+
+	// The Double Sign Jail period ends at Max Time supported by Amino
+	// (Dec 31, 9999 - 23:59:59 GMT).
+	DoubleSignJailEndTime = time.Unix(253402300799, 0)
```

Comment:
Shouldn't this be a parameter as well?

Reply:
This is because validators that equivocate are tombstoned and as a result are technically jailed "forever" -- hence it doesn't make sense for this to be configurable. i.e. it must be the maximum time allowed. If we ever remove tombstoning, then it can be configurable.